go/README.md

```diff
@@ -39,4 +39,9 @@
   ```
 
 - Fill out any missing variables
-  - Create a [Datahub Account](https://datahub-beta.figment.io/signup) for a free API key
\ No newline at end of file
+  - Create a [Datahub Account](https://datahub-beta.figment.io/signup) for a free API key
+
+- Go to `unchained/node` and install dependencies by running `yarn` (which will also prepare the git  pre-commit hook with `goimports`)
+  ```sh
+  yarn
+  ```
```

Comment:
I should have specified better, but I think this would be better suited in the top level readme (or could maybe do both) just so people don't accidentally miss if they are just quickly glancing at the top level readme. Can add to the `Setup` section